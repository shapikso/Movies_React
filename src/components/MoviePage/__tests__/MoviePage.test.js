// import React from 'react';
// import {shallow, mount} from 'enzyme';
// import MoviePage from "../MoviePage";
//
// describe('MoviePage', () => {
//     const props = {
//         genres: ['Animation', 'Family'],
//     };
//     it('should render correctly', () => {
//         const component = shallow(<MoviePage {...props} />);
//         expect(component).toMatchSnapshot();
//     });
//     it('should render 2 props from array', () => {
//         const component = mount(<MoviePage {...props} />);
//         expect(component.find('div div')).toHaveLength(props.genres.length);
//     });
//     it('should`t render anything ', () => {
//         const component = mount(<MoviePage {...props} genres={[]}/>);
//         expect(component.find('div div')).toHaveLength(0);
//     });
// });