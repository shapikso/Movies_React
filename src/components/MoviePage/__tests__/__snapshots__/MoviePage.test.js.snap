// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePage should render correctly 1`] = `ShallowWrapper {}`;
